portfolio/src/main/webapp/script.js

```diff
@@ -60,8 +60,9 @@ function userCommentsAsList(name, comments) {
 
 function createElementFrom(comment) {
   const liElement = document.createElement('dd');
-  //formats comment string and corresponding checkbox to be one line
-  liElement.innerText = comment.propertyMap.value.concat("    ");
+  const line = comment.propertyMap.value.concat('  (', comment.propertyMap.sentiment, ')');
+  //formats comment line and corresponding checkbox to be one line
+  liElement.innerText = line.concat('     '); 
```

Comment:
optional: You can consider using the ${} syntax to make string formatting slightly more compact

Reply:
Ohhh, I realize was bothered by all the concat() I will probably beautify those functions then after I finish up with the week 5 tutorial.